tests/test_pipe_viz.py

```diff
@@ -72,3 +76,21 @@ def test_png(viz_pipeline: Pipeline, tmp_path):
     # Verify that the output file exists and is a valid png file
     assert_path_exists(viz_file)
     assert imghdr.what(viz_file) == 'png'
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize("rankdir", RANKDIR_CHOICES)
+def test_from_cli(tmp_path: str, dataset_pandas: DatasetManager, rankdir: str):
```

Comment:
Is this testing anything other than "can morpheus cli accept the `viz_direction` argument?" We're not actually testing that the viz output is different though. I guess we don't have the infrastructure for that.

Reply:
Correct, it is testing that it can be run with all possible values, and still produce a result that is a valid PNG file.